front/app/containers/Admin/projects/project/events/edit.tsx

```diff
@@ -390,6 +449,35 @@ const AdminProjectEventEdit = () => {
       });
   };
 
+  // Intercepts API error responses and ensures a consistent error message is shown for maximum_attendees validation.
+  // Provides immediate feedback when a value is too low during input, matching the same error message shown
+  // if maximum_attendees validation fails during form submission.
+  const handleApiErrors = (apiErrors: ApiErrorType) => {
+    setSaving(false);
+
+    const maximumAttendeesErrorMessage = {
+      error: formatMessage(messages.maximumAttendeesError),
+    };
+
+    if (isError(apiErrors)) {
+      // It's a regular Error object - set a generic error
+      setErrors({
+        form: [{ error: formatMessage(messages.saveErrorMessage) }],
+      });
+    } else {
+      const errorObject = 'errors' in apiErrors ? apiErrors.errors : apiErrors;
+      const customErrors = { ...errorObject };
+
+      if ('maximum_attendees' in customErrors) {
+        customErrors.maximum_attendees = [maximumAttendeesErrorMessage];
+      }
+
+      setErrors(customErrors);
+    }
+
+    setSubmitState('error');
+  };
```

Comment:
Rolled my own approach here, so it's probably not how we conventionally do this.

Unfortunately, I haven't had time to check for an example of this on other forms yet.

I note that we don't instantly show errors in the form for other cases (e.g. Attendance button URL invalid), but only on save, so maybe the form as awhole needs some work?